### DISCUSSION TITLE
"react": "^18.2.0","tailwindcss": "^3.2.1",No prompt in webstore

### DISCUSSION THREAD

**Discussion by imyuanli** [tailwindlabs/tailwindcss#9701] [Help]
"react": "^18.2.0"
,"tailwindcss": "^3.2.1",
"postcss": "^8.4.18",
No prompt in webstore，use this npm install - D tailwindcss@npm : @ tailwindcss/postcss7-compat postcss @ ^ 7 autopresser @ ^ 9 command is still useless

**Comment by erikbg7** [comment]
Hey!
It is not clear - at least for me -  which is the issue, the steps to reproduce it, or the question.
Could you elaborate a little bit more the explanation so we can help you?
Thanks!
